genie-agent/src/test/groovy/com/netflix/genie/agent/execution/process/impl/JobProcessManagerImplSpec.groovy

```diff
@@ -546,4 +546,45 @@ class JobProcessManagerImplSpec extends Specification {
         cleanup:
         threadPoolScheduler.shutdown()
     }
+
+    def "Force kill diehard process"() {
```

Comment:
I can't really comment on the commit directly but the commit message is missing a space